src/lighteval/models/litellm_model.py

```diff
@@ -91,7 +91,7 @@ def __init__(self, config, env_config) -> None:
         self._tokenizer = encode
         self.pairwise_tokenization = False
         litellm.drop_params = True
-        litellm.verbose = True
+        litellm.set_verbose = False
```

Comment:
is the `verbose` vs `set_verbose` a typo on our end or does it depend on the litellm version?

Reply:
I assume it is the litellm version. I suggest litellm 1.50 since in later versions together_ai models experience issues.